tools/osbuilder/rootfs-builder/ubuntu/config.sh

```diff
@@ -12,13 +12,13 @@ PACKAGES="chrony iptables dbus"
 [ "$AGENT_INIT" = no ] && PACKAGES+=" init"
 [ "$MEASURED_ROOTFS" = yes ] && PACKAGES+=" cryptsetup-bin e2fsprogs"
 [ "$SECCOMP" = yes ] && PACKAGES+=" libseccomp2"
-REPO_URL=http://ports.ubuntu.com
+REPO_URL=${REPO_URL:-http://ports.ubuntu.com}
```

Comment:
Same comment on this PR about being able to set this and get this used via `make rootfs-image-tarball`